idaes/generic_models/unit_models/heat_exchanger.py

```diff
@@ -137,13 +149,64 @@ def delta_temperature_lmtd_callback(b):
     def delta_temperature(b, t):
         return (dT1[t] - dT2[t]) / log(dT1[t] / dT2[t])
 
+def delta_temperature_lmtd2_callback(b):
+    r"""
+    This is a callback for a temperature difference expression to calculate
+    :math:`\Delta T` in the heat exchanger model using log-mean temperature
+    difference (LMTD).  It can be supplied to "delta_temperature_callback"
+    HeatExchanger configuration option. This form is
+
+    .. math::
 
-def delta_temperature_amtd_callback(b):
+        \Delta T = \frac{\Delta T_2 - \Delta T_1}{
+            \log_e\left(\frac{\Delta T_2}{\Delta T_1}\right)}
+
+    where :math:`\Delta T_1` is the temperature differnce at the hot inlet end
```

Comment:
Typo: difference

This one appears a few times.

Reply:
Dang-old copy paste